Mathlib/Combinatorics/SimpleGraph/Copy.lean

```diff
@@ -229,4 +279,51 @@ lemma free_bot (h : A ≠ ⊥) : A.Free (⊥ : SimpleGraph β) := by
 
 end Free
 
+/-!
+#### Induced containment
+
+A graph `H` *inducingly contains* a graph `G` if there is some graph embedding `G ↪ H`. This amounts
+to `H` having an induced subgraph isomorphic to `G`.
+
+We denote "`G` is contained in `H`" by `G ⊴ H` (`\triangle_left_eq`).
+-/
+
+/-- A simple graph `G` is contained in a simple graph `H` if there exists an induced subgraph of `H`
+isomorphic to `G`. This is denoted by `G ⊴ H`. -/
+def IsIndContained (G : SimpleGraph V) (H : SimpleGraph β) : Prop := Nonempty (G ↪g H)
+
+@[inherit_doc] scoped infixl:50 " ⊴ " => SimpleGraph.IsIndContained
+
+protected lemma IsIndContained.isContained : G₁ ⊴ G₂ → G₁ ⊑ G₂ := fun ⟨f⟩ ↦ ⟨f, f.injective⟩
+protected lemma Iso.isIndContained (e : G ≃g H) : G ⊴ H := ⟨e⟩
+protected lemma Iso.isIndContained' (e : G ≃g H) : H ⊴ G := e.symm.isIndContained
+
+protected lemma Subgraph.IsInduced.isIndContained {G' : G.Subgraph} (hG' : G'.IsInduced) :
+    G'.coe ⊴ G :=
+  ⟨{  toFun := (↑)
+      inj' := Subtype.coe_injective
+      map_rel_iff' := hG'.adj.symm }⟩
+
+@[refl] lemma IsIndContained.refl (G : SimpleGraph V) : G ⊴ G := ⟨Embedding.refl⟩
+lemma IsIndContained.rfl : G ⊴ G := .refl _
+lemma IsIndContained.trans : G ⊴ H → H ⊴ I → G ⊴ I := fun ⟨f⟩ ⟨g⟩ ↦ ⟨g.comp f⟩
+
+lemma IsIndContained.of_isEmpty [IsEmpty V] : G ⊴ H :=
+  ⟨{  toFun := isEmptyElim
+      inj' := isEmptyElim
+      map_rel_iff' := fun {a} ↦ isEmptyElim a }⟩
+
+lemma isIndContained_iff_exists_iso_subgraph :
+    G ⊴ H ↔ ∃ (H' : H.Subgraph) (_e : G ≃g H'.coe), H'.IsInduced := by
```

Comment:
as before, can we have an equiv from graph embeddings to induced subgraphs of H iso to G